README.md

```diff
@@ -13,11 +13,12 @@ Included are 3 sample integrations:
 ## Getting Started
 
 1. Clone the project on to your local machine to get started
-2. Build and run in Simulator
-3. Use Charles proxy or Safari Web Inspector to see what events OM SDK is receiving
-4. Choose one of the sample implementations from the table
-5. Wait for the ad to load
-6. Observe network calls to `http://iabtechlab.com:66` (the query string will include URL-encoded JSON object with OM SDK event parameters)
+2. Download the latest omsdk-v1.js release from the IAB portal (https://tools.iab.techlab.com/omsdk) and drop it into //OMSDK/Service/
```

Comment:
What is the significance of the // in `//OMSDK/Service/`?  I don't recognize that for Github or Xcode development.

Reply:
Its just to denote the root of the GitHub directory.

Reply:
It might be :-)  I'll let @jwhines decide if it makes sense to non-Google developers.

Reply:
@juo007 One "/" is fine in denoting the root. I think two slashes is a Google thing.